forum/investors/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from startups.models import Location
+import uuid
+
+
+
+class Investor(models.Model):
+    """
+    Model representing an investor.
+
+    Attributes:
+        user (ForeignKey): The user associated with the investor.
+        company_name (str): The name of the investor's company.
+        available_funds (decimal): Available funds for investment.
+        experience_years (int): The years of experience of the investor.
+        location (ForeignKey): The location associated with the investor.
+        company_logo (file): Optional company logo.
+        description (str): Description of the investor's company.
+        website (str): The website of the company.
+        created_at (datetime): Date when the investor was created.
+    """
```

Comment:
The `docstring` is well written, but you could mention that the `user` field is a `ForeignKey` pointing to a user model, and clarify what happens when the `location` is set to null due to `SET_NULL` on deletion.

Reply:
thanks!